c2cgeoportal/scaffolds/create/deploy/hooks/pre-restore-database.mako

```diff
@@ -11,23 +11,26 @@ DATABASES=$@
 # databases / tables restore.
 sudo apache2ctl stop
 
+if psql -lqt | cut -d \| -f 1 | grep -qw $DATABASES; then
```

Comment:
we need the --quiet for grep, otherwise the script will output the result of the grep (string), while we only need to know if it founds it or not (exit status)